app/cypress/utils/graphql-test-utils.js

```diff
@@ -0,0 +1,12 @@
+// Utility to match GraphQL mutation based on the operation name
+export const hasOperationName = (req, operationName) => {
+  const { body } = req;
+  return body.hasOwnProperty("id") && body.id === operationName;
+};
+
+// Alias mutation if operationName matches
+export const aliasMutation = (req, operationName) => {
```

Comment:
This one actually works for both Mutations and Queries
```suggestion
export const aliasOperation = (req, operationName) => {
```